src/React/doc/index.rst

```diff
@@ -89,9 +89,13 @@ For example:
     }
 
 
+
 .. code-block:: html+twig
 
     {# templates/home.html.twig #}
+    
+    {% block stylesheets %}{{ encore_entry_link_tags('app') }}{% endblock %}
+    {% block javascripts %}{{ encore_entry_script_tags('app') }}{% endblock %}
```

Comment:
You are right ! 
Thank you for the precision, I didn't check the generated base.html.twig yet as the tutorial didn't extend it.

I added more informations so new users can make it work by copy-pasting, and still understand why it's needed with a link to related Encore documentation.

Sorry for the huge quantity of commits, I have been fighting with DOCtor RST, and he won by saying there is an error on a file I didn't change, on a line that does not have the problematic code : 

src/LiveComponent/doc/index.rst ✘
  581: Please use "html+twig" instead of "twig"


Have a good day !
Best regards

Reply:
Ok, let's see. Hmm. Earlier, on line 16 (first line under installation), we say:

> Before you start, make sure you have `Symfony UX configured in your app`_.

That is where, in theory, we're trying to point you to "make sure you have Encore fully installed and working in your app". By putting that note there, we don't need to include those details later on. Adding them here "muddies" the docs because these notes are really about setting up Encore/Symfony UX, and not about actually installing React.

Did you see that note on line 16? Is it too subtle? Could you suggest a change to it? I'd very much like to improve that as needed, and not include the new notes here (and, btw, absolutely no problem with many commits - keep going - we automatically squash when we merge).

Reply:
> we're trying to point you to "make sure you have Encore fully installed and working in your app"

I noticed that after, that's a bit of  "recursion"  but with attentive read it's clear enough.

> these notes are really about setting up Encore/Symfony UX, and not about actually installing React

I agree with you that this page's responsibility is about Symfony React UX and therefore don't need to remind every dependencies such as Encore on it. According to that, I removed the Encore note and just kept the modified example extending base.html.twig.